inline.h

```diff
@@ -2313,12 +2313,8 @@ Perl_cx_popformat(pTHX_ PERL_CONTEXT *cx)
 
 
 PERL_STATIC_INLINE void
-Perl_cx_pusheval(pTHX_ PERL_CONTEXT *cx, OP *retop, SV *namesv)
+S_push_evalortry_common(pTHX_ PERL_CONTEXT *cx, OP *retop, SV *namesv)
```

Comment:
inline functions (at least those in inline.h and visible to all XS) should use a `Perl_` prefix rather than the `S_` prefix

Reply:
Fixed in 9b080fb469